packages/framework/src/Model/Cart/CartMigrationFacade.php

```diff
@@ -88,6 +88,11 @@ public function mergeCurrentCartWithCart(Cart $cart): void
      */
     public function onKernelController(ControllerEvent $event): void
     {
+        $controller = $event->getController();
+        $frontendApiControllerName = 'Shopsys\FrontendApiBundle\Controller\FrontendApiController';
```

Comment:
why not `FrontendApiController::class`?

Reply:
Because the API is not installed in the framework, project-base has the power to install and uninstall it so I can not be sure whether the class is available :confused: